### ISSUE TITLE
Investigate:  "This is a private video. Please sign in to verify that you may see it" bug

### ISSUE THREAD

**Issue by zeeshanakram3** [Joystream/youtube-synch#82]
While syncing newly created video on YT, the syncing Lambda function logged the following error,

```
{
    "errorType": "Error",
    "errorMessage": "This is a private video. Please sign in to verify that you may see it.",
    "stack": [
        "Error: This is a private video. Please sign in to verify that you may see it.",
        "    at privateVideoError (/var/task/webpack:/youtube-sync/node_modules/ytdl-core/lib/info.js:109:12)",
        "    at validate (/var/task/webpack:/youtube-sync/node_modules/ytdl-core/lib/info.js:61:22)",
        "    at pipeline (/var/task/webpack:/youtube-sync/node_modules/ytdl-core/lib/info.js:184:11)",
        "    at process.info (internal/process/task_queues.js:95:5)",
        "    at exports.getBasicInfo (/var/task/webpack:/youtube-sync/node_modules/ytdl-core/lib/info.js:69:7)"
    ]
}
```

**Comment by dmtrjsg** [comment]
when there is a new video on YT and our syncing system starts pulling it in, the processing on YT still going. 

To fix this we need to add some sort of the delay to fetch them at a later point in time.

**Comment by zeeshanakram3** [comment]
Addressed in #91
